### ISSUE TITLE
Mdadm raid 1 /boot when in use

### ISSUE THREAD

**Issue by redmop** [dyindude/ubuntu-zfs#28]
Otherwise, the system will be unbootable if the first disk is lost.

**Comment by dyindude** [comment]
Yeah, that's a good point. I wasn't sure if I'd be using LUKS in my own usage when I started. I am leaning towards yes, so I'll take a look at this next time I get to coding

Thanks for the feedback!

**Comment by dyindude** [comment]
this has been implemented/merged to the master branch, if you're interested in trying it out, let me know what you think. otherwise I'll close this issue in a few days
